src/TLuaInterpreter.cpp

```diff
@@ -12410,6 +12468,22 @@ int TLuaInterpreter::startTempTrigger(const QString& regex, const QString& funct
     return id;
 }
 
+int TLuaInterpreter::startTempPromptTrigger(const QString& function)
+{
+    TTrigger* pT;
+    QStringList sList = {QString()};
+    QList<int> propertyList = {REGEX_PROMPT};
```

Comment:
Isn't this cool? We can have it all in one line thanks to C++11's initialiser lists.

`clang-format` makes the formatting a bit claustrophobic though, I would have preferred some space inside the braces.